apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx

```diff
@@ -129,6 +128,56 @@ describe("DepartureTimes", () => {
     }
   );
 
+  it("should display the detour badge with times if detour alert is present", () => {
```

Comment:
suggestion: consider adding a test for when there is both a high priority & detour alert to test the logic that only one would be shown